ruby/lib/clone_handler.rb

```diff
@@ -33,7 +33,9 @@ def response_body
   end
 
   def clone_command
-    @clone_command ||= response_body&.map { |info| "git clone #{info['clone_url']}" }&.join(' & ')
+    @clone_command ||= response_body&.map do |info|
+      "git clone #{info['clone_url']} #{username}/#{info['name']}"
```

Comment:
what I am understanding is that this
1. creates containing the folder if it does not exist
2. Is able to clone other repos to the same containing folder at the same time?

Maybe we need to confirm that these requests are happening in parallel.

Recently in the bash solution I switched the git clone from 

`git clone repo1.git repo2.git repo3.git repo4.git .. etc`
to

a bunch of jobs that run `git clone` independantly